lib/presentation/util/extensions/color_extension.dart

```diff
@@ -72,4 +72,5 @@ extension AppColor on Color {
   static const networkConnectionBackgroundColor = Color(0xff7B7B7B);
   static const searchBottomBarColor = Color(0xffF2F2F2);
   static const searchResultsCountTextColor = Color(0xffACACAC);
+  static const activeDarkGrayColor = Color(0xff070707);
```

Comment:
I don't think it is a good idea to define the color like that, we can not reuse and hard to change when some screen is change the concept

Reply:
The application has a color palette we don't have that many colors, I think having const color for each element is bad

Reply:
https://github.com/linagora/linshare-mobile-flutter-app/pull/282#discussion_r602997776

Reply:
Changed it to function